GeneratorInterface/Core/interface/ConcurrentGeneratorFilter.h

```diff
@@ -158,12 +158,13 @@ namespace edm {
     void initLumi(gen::GenStreamCache<HAD, DEC>* cache, LuminosityBlock const& index, EventSetup const& es) const;
     ParameterSet config_;
     mutable std::atomic<gen::GenStreamCache<HAD, DEC>*> useInLumi_{nullptr};
-    mutable std::atomic<unsigned long long> greatestNStreamEndLumis_{0};
+    mutable std::atomic<unsigned long long> nextNGlobalBeginLumis_{1};
     mutable std::atomic<bool> streamEndRunComplete_{true};
     // The next two data members are thread safe and can be safely mutable because
```

Comment:
Should this become
```suggestion
    // The next three data members are thread safe and can be safely mutable because
```
?

Reply:
Yes. Fixed the comment in both files. Thanks. I will push the changes momentarily.